scala_proto/scala_proto.bzl

```diff
@@ -1,4 +1,5 @@
 load("//scala:scala.bzl",
+  "scala_mvn_artifact",
```

Comment:
how does this build? I thought you moved this to a new file?

Reply:
I was puzzled by this too. It looks like if you load a symbol in `bzl` files, it's available to both you and downstream `bzl` files. Not sure if it's intended that way.